Transbank/Onepay/Utils/OnepayRequestBuilder.cs

```diff
@@ -29,14 +33,22 @@ private long GetTicksNow()
             return (long)DateTime.Now.Ticks / TimeSpan.TicksPerMillisecond;
         }
 
-        public SendTransactionRequest BuildSendTransactionRequest(ShoppingCart cart, ChannelType channel,
-            Options options)
+        public SendTransactionRequest BuildSendTransactionRequest(ShoppingCart cart, string externalUniqueNumber,
```

Comment:
This is the same parameter order we used on the java sdk, right?

Reply:
No, they are not the same

Reply:
And why not?

Reply:
I didn't realize. Strictly, users will call Transaction.create and not OnepayRequestBuilder

Reply:
That's right, but I'd bet that in the future it will bite us and confuse another dev or our own future selves. 

Unless there is a good reason to have different ordering, I'd keep it consistent across languages (and consistent with `Transaction.create` too)